src/greensenti/dhus.py

```diff
@@ -13,12 +13,13 @@
     from google.cloud import storage
 except ImportError:
     GCLOUD_DISABLED = True
+    storage = None
```

Comment:
This is to avoid warnings from ruff?

Reply:
Imports in the try block should also be defined in the except block (failing to do so can lead to issues with static checkers, which may not recognize the import as being properly defined).